lib/ReactViews/StandardUserInterface/StandardUserInterface.jsx

```diff
@@ -47,6 +47,17 @@ export const showStoryPrompt = (viewState, terria) => {
     terria.stories.length === 0 &&
     viewState.toggleFeaturePrompt("story", true);
 };
+const GlobalTerriaStyles = createGlobalStyle`
+  ${props =>
+    props.experimentalFeatures &&
+    `
+    body {
+      *:focus {
+        outline: 3px solid #C390F9;
```

Comment:
Just wondering, was there a reason why the purple colour was used when clicking on search-related things?

Reply:
Ripped this straight from the DTA design system!

Reply:
It's not just search-related things, it's anything in the UI that can be focused - currently we have no outlines on anything and I think that's just a dang shame 🙁 (so I've snuck it in like this for anything that has experimentalFeatures turned on)